apps/web/src/data/subgraph/requests/dashboardQuery.ts

```diff
@@ -0,0 +1,43 @@
+import * as Sentry from '@sentry/nextjs'
+
+import { PUBLIC_DEFAULT_CHAINS } from 'src/constants/defaultChains'
+import { SDK } from 'src/data/subgraph/client'
+
+export const dashboardRequest = async (memberAddress: string) => {
+  if (!memberAddress) throw new Error('No user address provided')
```

Comment:
perhaps we should add this inside the try catch block so that the FE doesn't crash if this condition is met / we sentry log the error

Reply:
Yup. Definitely